actors/builtin/storage_market/storage_market_actor_state.go

```diff
@@ -26,7 +26,8 @@ type PartyDeals struct {
 }
 
 type StorageMarketActorState struct {
-	Deals cid.Cid // AMT[]OnChainDeal
+	Deals cid.Cid // AMT[DealID]DealProposal
```

Comment:
Since renaming the value type to DealProposal, call this `Proposals`, which will also make a bunch of the `proposals := AsDealArray()` variables above make more sense.

Alternatively, keep this as `Deals` but rename those variables to match.